client/src/app/site/pages/organization/pages/committees/modules/committee-meeting-preview/committee-meeting-preview.component.ts

```diff
@@ -100,10 +100,26 @@ export class CommitteeMeetingPreviewComponent {
     }
 
     public async toggleTemplateMeeting(): Promise<void> {
+        const content = this.title;
+
         if (this.isTemplateMeeting) {
-            await this.meetingRepo.update({ set_as_template: false }, { meeting: this.meeting });
+            const title = this.translate.instant(
+                `Do you really want to stop sharing this meeting as a public template?`
+            );
+
+            const confirmed = await this.promptService.open(title, content);
+            if (confirmed) {
+                await this.meetingRepo.update({ set_as_template: false }, { meeting: this.meeting });
+            }
         } else {
-            await this.meetingRepo.update({ set_as_template: true }, { meeting: this.meeting });
+            const title = this.translate.instant(
+                `Should this meeting really be made available as a public template? Meeting templates and the data they contain are publicly viewable by all committee administrators.`
```

Comment:
This is probably too long for a title, maybe you should put the explanation part in the content.

Reply:
![Bildschirmfoto_2022-08-17_13-29-58](https://user-images.githubusercontent.com/9317999/185107899-619c2e90-f4f3-4766-8703-f1eeb0c2cb94.png)